subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1667,6 +1683,15 @@ static int init_reset(void)
 	mem_init(conn_pool, sizeof(struct ll_conn),
 		 sizeof(conn_pool) / sizeof(struct ll_conn), &conn_free);
 
+	/* Invalidate connection handles, refer to ll_connected_get() */
+	for (uint16_t handle = 0U; handle < CONFIG_BT_MAX_CONN; handle++) {
```

Comment:
```suggestion
	for (int handle = 0U; handle < CONFIG_BT_MAX_CONN; handle++) {
```

Suggest to use same type for the iterator and the value compared to

Reply:
Fair :) We don't have a rule for this IIRC